lib/seed_utils.rb

```diff
@@ -33,6 +68,11 @@ def seed_table_with_upload(klass, user, options = {})
     puts 'Done!'
   end
 
+  def load_table(klass, _user, _file_options, csv_name)
```

Comment:
Is _user and _file_options used here?

Reply:
I will be now, I'm breaking part of the code out into a new function.